org.osate.aadl2.errormodel.faulttree.generation/META-INF/MANIFEST.MF

```diff
@@ -16,8 +16,7 @@ Require-Bundle: org.eclipse.ui,
  org.eclipse.emf.transaction;bundle-version="1.9.0",
  org.eclipse.sirius.ext.base;bundle-version="5.0.1",
  org.eclipse.ui.browser;bundle-version="3.6.100",
- org.eclipse.ui.ide;bundle-version="3.13.1",
- org.eclipse.sirius.common.ui;bundle-version="5.1.1"
```

Comment:
Why didn't you just lower the dependency to 5.1.0 and leave the code calling SiriusTransPlugin in?